app/javascript/controllers/add_condition_controller.js

```diff
@@ -0,0 +1,50 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  addCondition(event) {
+    event.preventDefault()
+    const form = document.getElementById("add-condition-form")
+    const clone = form.cloneNode(true)
+    const length = document.querySelectorAll(".condition-form").length
+    this.changeIds(clone, length)
+    clone.id = `add-condition-form-${length}`
+    clone.classList.remove("display-none")
+    const place = document.getElementById("new-forms")
+    place.appendChild(clone)
+  }
+
+  removeCondition(event) {
+    event.preventDefault()
+    event.srcElement.parentNode.remove()
+  }
+
+  changeIds(form, length) {
+    const inputs = form.getElementsByTagName("input")
+
+    for (let a = 0; a < inputs.length; a++) {
+      const formCount = 3 + length
+      inputs[a].name = inputs[a].name.replace(/\d/g, `${formCount}`)
+      inputs[a].id = inputs[a].id.replace(/\d/g, `${formCount}`)
+    }
```

Comment:
A `for..of` loop might be clearer here:

```suggestion
    for (const input of inputs) {
      const formCount = 3 + length
      input.name = input.name.replace(/\d/g, `${formCount}`)
      input.id = input.id.replace(/\d/g, `${formCount}`)
    }
```